### ISSUE TITLE
Merge Uber Traffic integration

### ISSUE THREAD

**Issue by MichaelsJP** [GIScience/openrouteservice#1386]
Waiting for #1359

**Comment by TheGreatRefrigerator** [comment]
- should be working with the changes in the backend now

**Comment by TheGreatRefrigerator** [comment]
https://github.com/GIScience/openrouteservice/tree/add_uber_traffic ?

**Comment by MichaelsJP** [comment]
Yeah. Should be it.

> This branch is [11 commits ahead](https://github.com/GIScience/openrouteservice/compare/master...add_uber_traffic), [922 commits behind](https://github.com/GIScience/openrouteservice/compare/add_uber_traffic...master) master.

:hear_no_evil:

**Comment by redfrexx** [comment]
We have decided to leave it as an experimental implementation and not merge this code into master for now. Instead, it would make more sense to come up with a software architecture which properly integrates the classes used for processing the here and uber data to reduce code duplication. 

We have created a new branch 'implement_uber_traffic' based on the current master and added all the classes from the old uber traffic integration. The code builds, tests are running, and routes are generated. But definitely needs major refactoring and testing. So **this is not meant to be merged into master**, just to have a running version of the uber traffic integration using the ORS code after the GH update for later reference when reworking the traffic integration in general.
